plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_create_release_ctrl.js

```diff
@@ -75,10 +76,12 @@ samson.controller('KubernetesCreateReleaseCtrl',
     };
 
     $scope.toggleAll = function() {
-      var newState = !$scope.allToggled();
-      _.each($scope.deploy_groups, function(deploy_group) {
-        deploy_group.selected = newState;
-      });
+      if(!$scope.submitting) {
```

Comment:
Unfortunately, disabling a link, i.e., setting the `disabled` attribute, is still not a standard across browsers, thus, not supported is some browser:

http://stackoverflow.com/questions/10276133/how-to-disable-html-links

I did try to disable it by setting the attribute `disabled`, but it didn't work, so, the workaround was to prevent the action from happening in the javascript code.

However, I should have greyed out the text when the form is disabled, but forgot to do it. I'll create a separate PR for taking care of the styling part of it.

Thanks for the heads up.